### ISSUE TITLE
Can't play video

### ISSUE THREAD

**Issue by h3poteto** [h3poteto/fedistar#1293]
```
KMS: DRM_IOCTL_MODE_CREATE_DUMB failed: Permission denied
Failed to get GBM buffer from swap chain: error creating plane 0 of size 818x374 and format 538982482: Permission denied
```

Similar to https://github.com/tauri-apps/tauri/issues/8462
Even though I use `WEBKIT_DISABLE_DMABUF_RENDERER=1`, it still can't play video.

```
extra/webkit2gtk-4.1 2.42.4-1 (27.4 MiB 108.0 MiB) (Installed)
    Web content engine for GTK
extra/webkit2gtk 2.42.4-1 (27.4 MiB 107.9 MiB) (Installed)
    Web content engine for GTK
```

**Comment by h3poteto** [comment]
refs https://github.com/tauri-apps/tauri/issues/8579

**Comment by h3poteto** [comment]
This has been resolved with 1.9.10.
